lib/build.gradle

```diff
@@ -4,7 +4,7 @@ android {
     compileSdkVersion 28
 
     defaultConfig {
-        minSdkVersion 14
+        minSdkVersion 18
```

Comment:
This attribute needs to remain 14, and a `PreProcessException` should be thrown inside the preprocess step if the version isn't 18 and up, eg, `if (android.os.Build.VERSION.SDK_INT >= ...`